exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlConverter.java

```diff
@@ -139,6 +151,24 @@ public SqlNode parse(String sql) {
       SqlParser parser = SqlParser.create(sql, parserConfig);
       return parser.parseStmt();
     } catch (SqlParseException e) {
+
+      // Attempt to use default back_tick quote character for identifiers when
```

Comment:
Why would you switch to back_tick quoting, even when option ANSI_QUOTES is on? Why would you want to try with a configuration different from option set by user?

also, sql.contains("`") may not work, if the sql string contains ` as a character in a string literal. I could not see how this block of code will work for such case.